scripts/e2e-tests.sh

```diff
@@ -314,6 +314,8 @@ function setup_test_cluster() {
   set +o pipefail
 
   if (( ! SKIP_KNATIVE_SETUP )) && function_exists knative_setup; then
+    echo "Waiting for Istio installation to complete before starting knative_setup"
```

Comment:
```suggestion
    # Wait for Istio installation to complete, if necessary, before calling knative_setup.
```

Why?
1. Because we don't wait if Istio add-on is not installed.
1. wait_until_batch_job_complete() is verbose enough.

Reply:
Thanks for the explanation :)